cmd/embedded-cluster/reset.go

```diff
@@ -169,66 +201,85 @@ func newHostInfo(ctx context.Context) (hostInfo, error) {
 	if err != nil {
 		return currentHost, err
 	}
+	currentHost.RoleName = "worker"
 	// control plane only stff
-	if currentHost.isControlPlane() {
+	if currentHost.Status.Role == "controller" {
+		currentHost.RoleName = "controler"
 		// fetch controlNode
 		err := currentHost.getControlNodeObject(ctx)
 		if err != nil {
 			return currentHost, err
 		}
 	}
+	// try and get custom role name from the node labels
+	labels := currentHost.Node.GetLabels()
+	if value, ok := labels["kots.io/embedded-cluster-role-0"]; ok {
+		currentHost.RoleName = value
+	}
 	return currentHost, nil
 }
 
-func checkErrPrompt(err error) bool {
+func checkErrPrompt(c *cli.Context, err error) bool {
 	if err == nil {
 		return true
 	}
 	fmt.Println("-----")
 	fmt.Println(err)
 	fmt.Println("-----")
-	fmt.Println("An error has occured while trying to reset this node.")
-	fmt.Println("Continuing may leave the cluster in an unexpected state")
+	fmt.Println("An error occurred while trying to reset this node.")
+	fmt.Println("Continuing may leave the cluster in an unexpected state.")
+	if c.Bool("no-prompt") {
+		return true
+	}
 	return prompts.New().Confirm("Do you want to continue anyway?", false)
 }
 
 var resetCommand = &cli.Command{
 	Name: "reset",
 	Flags: []cli.Flag{
 		&cli.BoolFlag{
-			Name:   "yes-really-reset",
+			Name:   "confirm",
```

Comment:
How come this one and force are hidden?

Reply:
they're hidden because they're potentially dangerous and exist as break-glass flags. 

by making them visible in the help text a user might attempt to run with them by default and skip any pre-reset checks

Reply:
thinking about it, maybe `--force` should be visible but explain the difference in a prompt

Reply:
https://github.com/replicatedhq/embedded-cluster/pull/281/commits/2dae3c18280705dddc4de229441b6ed05dbe27f0